glean-core/src/lib.rs

```diff
@@ -142,6 +145,32 @@ impl Glean {
         Ok(glean)
     }
 
+    /// Create and initialize a new Glean object.
+    ///
+    /// This will create the necessary directories and files in `data_path`.
```

Comment:
And it clears any existing database, right?  That seems potentially surprising and should probably be called out.